php-checks/src/test/resources/checks/MultilineBlocksCurlyBracesCheck.php

```diff
@@ -2,56 +2,90 @@
 
 function nonCompliant() {
 
+  if ($condition) firstActionInBlock();
+
+  if ($condition) firstActionInBlock();  secondAction();  // Noncompliant
+  //              ^^^^^^^^^^^^^^^^^^^^^> ^^^^^^^^^^^^^^^
+
+  if ($condition);  secondAction();  // Noncompliant
+  //             ^> ^^^^^^^^^^^^^^^
+
+  if ($condition) firstActionInBlock();
+  secondAction();
+
+  if ($condition) firstActionInBlock();
+    secondAction();  // Noncompliant
+
+  if ($condition)
+    firstActionInBlock();
```

Comment:
I'm not sure that highlighting the first line in multi-line cases is a nice choice. We will end up with everything highlighted, and it will be a mess to sort things up.
I would prefer to highlight only the statements following the first one raising the issue. Ignoring the first one (correctly placed)